tests/virt_autotest/login_console.pm

```diff
@@ -35,6 +35,7 @@ sub login_to_console() {
     assert_screen([qw(grub2 grub1)], 120);
     if (!get_var("reboot_for_upgrade_step")) {
         if (get_var("XEN") || check_var("HOST_HYPERVISOR", "xen")) {
+            send_key 'up';
```

Comment:
To the comment for line 472, the system_role.pm should be loaded for installation to sle12sp2 and its above. Without my change, the original code load this file only for test put in sles12sp2/sles12sp3 job group and without INSTALL_TO_OTHERS. This is not correct for our tests that are put in sles12sp3 job group, but install to sles12sp2 at first via INSTALL_TO_OTHERS.